### ISSUE TITLE
5.6.1 image file access is very slow on IDR

### ISSUE THREAD

**Issue by manics** [ome/omero-model#71]
OMERO.server 5.6.1 introduced a validation check to prevent malicious image files gaining access to other data: https://ome-www-dev.openmicroscopy.org/security/advisories/2019-SV1/
This check takes too long on the IDR- the readonly servers have a PostgreSQL statement timeout of 1 minute and the query is timing out.

Testing on the read-write server which has no timeout also takes a long time, I cancelled the request after a few minutes.

This takes less than a second to run on 5.6.0, output should be `(2048, 2169)`:
```python
#!/opt/omero/server/venv3/bin/python
from omero.gateway import BlitzGateway
conn = BlitzGateway(host='localhost', username='public', passwd='public', secure=True)
assert conn.connect()
im = conn.getObject('Image', 4007802)
px = im.getPrimaryPixels()
plane = px.getPlane(0, 0, 0)
print(plane.shape)
conn.close()
```
It times out on 5.6.1

**Comment by sbesson** [comment]
Retested while investigating the OMERO.server upgrade on IDR - see https://github.com/IDR/deployment/issues/327#issuecomment-830061842. After regenerating the (invalidated) Bio-Formats cache files, images are accessible as expected. The sample script in the description of this PR also returns promptly:

```
[sbesson@prod97-omeroreadwrite ~]$ cat test.py 
#!/opt/omero/server/venv3/bin/python
from omero.gateway import BlitzGateway
conn = BlitzGateway(host='localhost', username='public', passwd='public', secure=True)
assert conn.connect()
im = conn.getObject('Image', 4007802)
px = im.getPrimaryPixels()
plane = px.getPlane(0, 0, 0)
print(plane.shape)
conn.close()
[sbesson@prod97-omeroreadwrite ~]$ time /opt/omero/server/venv3/bin/python test.py 
(2048, 2169)

real	0m1.770s
user	0m1.635s
sys	0m1.697s
```

This seems to indicate the INDEX creation discussed above is not strictly necessary and there is no requirement for a DB upgrade in an upcoming OMERO.server release.

A secondary issue discovered while testing a selection of IDR datasets is that the fix for 2016-SV1 breaks existing assumptions for publishing IDR datasets especially in terms of data containment. I will keep testing but I would be inclined to close this (and the associated PRs) and open a separate issue.

**Comment by manics** [comment]
> This seems to indicate the INDEX creation discussed above is not strictly necessary and there is no requirement for a DB upgrade in an upcoming OMERO.server release.

I'm pretty sure that's because @joshmoore manually created an postgres index to workaround it 😀

**Comment by sbesson** [comment]
Retested in https://github.com/ome/omero-model/pull/72#issuecomment-921762200. I could not reproduce the behavior initially reported which might have been. The upgrade is still blocked by a separate issue.
